src/Uno.UI/UI/Xaml/Controls/Image/Image.wasm.cs

```diff
@@ -102,7 +105,28 @@ private void OnSourceChanged(DependencyPropertyChangedEventArgs e)
 
 			_lastMeasuredSize = _zeroSize;
 
-			if (source is WriteableBitmap wb)
+			var stream = source?.Stream;
+			if (stream != null)
+			{
+				stream.Seek(0, SeekOrigin.Begin);
+				var image = SixLabors.ImageSharp.Image.Load<Bgra32>(stream);
```

Comment:
Thanks for the PR! In general we try to avoid adding mandatory dependencies to Uno, and in this case it's probably not necessary.

For this instance, you'll need to build a data uri, and call `SetImageUrl(dataUri);`, and let the browser do the decoding. You probably won't be able to determine the format of the image, so maybe use `application/octet-stream` mime type will work.

Reply:
Done! Thanks for the info!